pyaedt/modules/Mesh.py

```diff
@@ -350,9 +350,9 @@ def assign_surface_mesh_manual(self, names, surf_dev=None, normal_dev=None, aspe
         ----------
         names : list
             List of faces to apply the surface mesh to.
```

Comment:
If we put a list of object it is working too.

Reply:
So the doc can be "List of faces or directly objects to apply the surface mesh to."?